src/js/jsx/sections/export/ExportAllPanel.jsx

```diff
@@ -40,7 +39,6 @@ define(function (require, exports, module) {
     var strings = require("i18n!nls/strings"),
         svgUtil = require("js/util/svg"),
         collection = require("js/util/collection"),
-        gridSort = require("js/util/gridsort"),
```

Comment:
@mcilroyc I think this change came when I rebased against master? I didn't delete gridSort so maybe @designedbyscience did for a reason? Then we can get more clarity about this and whether it is necessary or not, thus may not be an issue.